pkg/diag/validator/resource.go

```diff
@@ -40,6 +40,9 @@ func (r Resource) Namespace() string { return r.namespace }
 func (r Resource) Status() Status    { return r.status }
 func (r Resource) Error() error      { return r.err }
 func (r Resource) String() string {
+	if r.namespace == "default" {
+		return fmt.Sprintf("%s/%s", r.kind, r.name)
+	}
 	return fmt.Sprintf("{%s:%s/%s}", r.kind, r.namespace, r.name)
```

Comment:
This is different from Deployment.String()

Reply:
Thanks! I will double check.

Reply:
this is fixed in #4270